From: Giovanni Mascellani <mascellani@poisson.phc.unipi.it>
Date: Thu, 17 Feb 2011 11:07:04 +0100
Subject: Add classpath to JAR.

Instruct maven to add classpath information to the generated JARs.
---
 msv/pom.xml |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/msv/pom.xml b/msv/pom.xml
index d5a2f84..081eaf6 100644
--- a/msv/pom.xml
+++ b/msv/pom.xml
@@ -64,6 +64,7 @@ EVEN IF SUN HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
           <archive>
             <manifest>
               <mainClass>com.sun.msv.driver.textui.Driver</mainClass>
+              <addClasspath>true</addClasspath>
             </manifest>
           </archive>
         </configuration>
-- 
